mimic/rest/cinder_api.py

```diff
@@ -31,7 +31,14 @@ def catalog_entries(self, tenant_id):
         """
         return [
             Entry(
-                tenant_id, "volume", "cloudBlockStorage",
```

Comment:
Although, mimic's cinder fakery is already out of line with Rackspace since it only provides Cinder v2  API.